src/Mailer/Email.php

```diff
@@ -2061,6 +2061,7 @@ public function send($content = null)
         return $contents;
     }
 
+    
```

Comment:
I fixed up the code. This you need also to revert, please make sure you do not introduce noise whitespace.

Reply:
Ok thank you for all the input